crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -364,9 +364,7 @@ impl<'w, 's> Commands<'w, 's> {
     /// - [`spawn_batch`](Self::spawn_batch) to spawn entities with a bundle each.
     #[track_caller]
     pub fn spawn<T: Bundle>(&mut self, bundle: T) -> EntityCommands {
```

Comment:
Why is this the only method (as far as I can see at a first glance) that still uses `&mut self`?

Reply:
That's a `Commands` method, not an `EntityCommands` method. That caught me at first as well.

Reply:
Not OP, but my guess is that you likely don't want to consume the whole `Commands`. Other `Commands` methods are also unchanged, it seems the changes are only for `EntityCommands`

Reply:
Whoops, you're totally right @Luminoth :D